src/main/java/edu/harvard/iq/dataverse/engine/command/impl/UpdatePermissionRootCommand.java

```diff
@@ -37,6 +37,13 @@ public Dataverse execute( final CommandContext ctxt) throws CommandException {
 		}
 	}
 
+    @Override
+    public boolean onSuccess(CommandContext ctxt, Object r) {  
+        return ctxt.dataverses().index((Dataverse) r,true);
+    }        
+
+
+    //TODO: Review this as this will never be an instance of Dataset, will it?
```

Comment:
This will never be a dataset so we can simplify the code here

Reply:
Similar to last, left as todo and out of scope.